packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/react/getuserinfo.md

```diff
@@ -85,13 +80,7 @@ The `authState` contains an `idToken` that contains the user profile claims. You
    export default Profile;
    ```
 
-3. Install `semantic-ui-react` into your app with the following command:
-
-   ```shell
-   npm install semantic-ui-react
-   ```
-
-4. Import `Profile` into `src/App.js` by adding the following:
+4. Import `Profile` into `src/App.jsx` by adding the following:
```

Comment:
I'll add text about adding a link to `/profile` similar to what we discussed for Angular